glide.yaml

```diff
@@ -12,3 +12,5 @@ import:
   version: master
 - package: github.com/giantswarm/operatorkit
   version: master
+- package: k8s.io/client-go
+  version: v3.0.0
```

Comment:
You should not need this since operatorkit defines all of this already.

Reply:
Yeah, removed.